ibis/backends/oracle/converter.py

```diff
@@ -12,7 +13,11 @@ def convert_Timestamp_element(cls, dtype):
 
     @classmethod
     def convert_Date_element(cls, dtype):
-        return datetime.date.fromisoformat
+        # TODO (mehmet): Not sure if we should modify this... Could
+        # not run the tests on this backend. If we do, should we also
+        # modify `convert_Timestamp_element()` and
+        # `convert_Time_element()` similarly?
+        return pd.Timestamp.fromisoformat
```

Comment:
This is sort of up for grabs given that pandas doesn't have a standard way to represent an array of dates (the `_element` suffix implies [perhaps not in an obvious way] that this function is being called once per element of an array).

I think it's fine to also change this to using pandas timestamps.